core-web/apps/dotcms-ui/src/app/view/components/dot-crumbtrail/service/dot-crumbtrail.service.ts

```diff
@@ -139,6 +146,28 @@ export class DotCrumbtrailService {
         );
     }
 
+    /**
+     * Get the pages crumbtrail.
+     * Alternate crumbtrail is used when the page portlet is not enabled.
+     *
+     * @private
+     * @param {DotCrumb[]} alternateCrumbTrail
+     * @return {*}  {Observable<DotCrumb[]>}
+     * @memberof DotCrumbtrailService
+     */
+    private getPagesCrumbTrail(alternateCrumbTrail: DotCrumb[] = []): Observable<DotCrumb[]> {
+        return this.getMenuLabel('pages').pipe(
```

Comment:
Can we add a `test` to ensure that a `getManuLabel('pages')` is called when the route is `browser/edit-page`?

Reply:
I'll take a look to see how can we make it

Reply:
Done!